src/calculateRentalCost.js

```diff
@@ -1,10 +1,13 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
```

Comment:
Again, use constants for the number of days and the discount amount to avoid hardcoding these values.